truestbyheart/politico
# [feature #163974247] added functionality to remove empty values

#### What does this PR do?

 It checks for empty values sent by the user.

#### Description of Task to be completed?

When the data is removed from the user the server checks if there is an empty value and if its present the server respond with the empty value so as to inform the user of its presence and if not it just stores it.

#### How should this be manually tested?

- Clone the repo to your local machine
- Run `npm install` to install the dependencies
- Once the dependencies are installed run `npm run start`
- Then open `postman` and post data to `/parties`
- Send empty values and check for the response
 
#### Any background context you want to provide?

 Before the update the server just received empty value and had no means of checking it.

#### What are the relevant pivotal tracker stories?
 #163974247
